Source/WebCore/platform/cocoa/PlaybackSessionModel.h

```diff
@@ -150,10 +150,16 @@ class PlaybackSessionModel : public CanMakeWeakPtr<PlaybackSessionModel> {
     virtual bool supportsLinearMediaPlayer() const { return false; }
 #endif
 
+    virtual bool prefersAutoDimming() const { return m_prefersAutoDimming; }
```

Comment:
have the default return true; rather than having the need for protected variable. We typically don't want those here ; they should be private